examples/cpp/Makefile

```
@@ -0,0 +1,21 @@
+CXX := /opt/wasi-sdk/bin/clang++
+CXXFLAGS := -O2 -I ../../include/
+WIZER := ../../target/release/wizer
+WASMTIME := wasmtime
+
+.PHONY: all
+all: main_initialized.wasm
+
+main.wasm: main.cpp
+	$(CXX) $(CXXFLAGS) -o $@ $^
+
+main_initialized.wasm: main.wasm
+	$(WIZER) -r _start=wizer.resume -o $@ $^
```

Needed for the suggested change to the cpp file below:

```suggestion
	$(WIZER) --allow-wasi -r _start=wizer.resume -o $@ $^
```

---

Done!